backend/routers/backup.py

```
@@ -75,22 +75,96 @@ def _save_backup_config(config: dict):
+        key = _get_encryption_key()
     key_bytes = key.encode()
     text_bytes = text.encode()
     encrypted = bytes([text_bytes[i] ^ key_bytes[i % len(key_bytes)] for i in range(len(text_bytes))])
-    return base64.b64encode(encrypted).decode()
+    return ENCRYPTION_VERSION + base64.b64encode(encrypted).decode()
 
 
-def _simple_decrypt(encrypted_text: str, key: str = "my-ledger-backup-key") -> str:
-    """简单解密"""
+def _simple_decrypt(encrypted_text: str, key: str | None = None) -> str:
+    """简单解密，支持当前密钥和历史密钥"""
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)